frame/support/src/traits/storage.rs

```diff
@@ -48,9 +48,13 @@ pub trait StorageInstance {
 	const STORAGE_PREFIX: &'static str;
 }
 
-/// Some info about an individual storage in a pallet.
+/// Metadata about storage from the runtime.
 #[derive(codec::Encode, codec::Decode, crate::RuntimeDebug, Eq, PartialEq, Clone)]
 pub struct StorageInfo {
+	/// Encoded string of pallet name.
+	pub pallet_name: Vec<u8>,
```

Comment:
This is the name of the pallet instance, correct?

Reply:
it should be the name given to the pallet in construct_runtime